### DISCUSSION TITLE
Speculations on GNU kernel future

### DISCUSSION THREAD

**Discussion by AnimusPEXUS** [rms-support-letter/rms-support-letter.github.io#4207] [General]
As I understand, there are no clear knowledge on what Torvalds thinks on all this Stallman situation. - as I understand, Torvalds is related to RedHat and being payed by it. so, I suppose, he shares RH's view.

I predict, It is possible, what Linux going to embrace Rust as It's main language little by little.

The question is: will Rust be integrated in GCC?, taking to account what Mozilla is pressing on GNU and FSF.

Also, I predict, it is possible what Linux will be switched to LLVM because of all what is going on with GNU.

In this case, possibly, it is good time for some _more safe language_-based GNU kernel project to appear, maybe on D.

By the way, is there known reaction from D community?

**Comment by AnimusPEXUS** [comment]
bump.

is there are any known Torvalds opinion on Stallman situation?

**Reply by 6r1d** [reply]
Not that I'm aware of.
https://twitter.com/linus__torvalds
